rs/ic_os/config/src/lib.rs

```diff
@@ -31,8 +74,170 @@ fn parse_config_line(line: &str) -> Option<(String, String)> {
 pub fn config_map_from_path(config_file_path: &Path) -> Result<ConfigMap> {
     let file_contents = read_to_string(config_file_path)
         .with_context(|| format!("Error reading file: {}", config_file_path.display()))?;
-    Ok(file_contents
+
+    let normalized_file_contents = file_contents.replace("\r\n", "\n").replace("\r", "\n");
+
```

Comment:
Before, we were normalizing the config.ini in the old config.sh script, but now we're doing it here before parsing.